packages/webapp/pages/onboarding.tsx

```diff
@@ -186,21 +215,19 @@ export function OnboardPage(): ReactElement {
   };
 
   const onClickCreateFeed = () => {
-    const onboardingChecklist = getFeatureValue(feature.onboardingChecklist);
```

Comment:
Since it's in onboarding it's always new users which makes it useless to check if they enrolled or not:
https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1729013039886069